api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -1361,30 +1367,56 @@ public void getUserByEmail_shouldGetUserByUsingEmail() {
 		User user = userService.getUserByEmail("hank.williams@gmail.com");
 		assertNotNull("User with email hank.williams@gmail not found in database", user);
 	}
-	/*
-	 * @see UserService#getUserByActivationKey(String)
-	 */
+	
 	@Test
-	public void getUserByActivationKey_shouldGetUserUsingActivationKey() {
-		executeDataSet(XML_FILENAME);
-		String key="XbklsieiskeuztecEnPf";
-		User user = userService.getUserByActivationKey(key);	
-		assertNotNull("User with activationKey ", user); 
-		String[] tokens = user.getActivationKey().split(":");
-		assertEquals(key,tokens[0]);		
-
+	public void verifyCreateUserActivationKey_shouldCreateUserActivationKey() {
```

Comment:
By convention the part of a test method before the underscore should be the name of the method you are testing